### PR TITLE
added tiggers for s annd t

### PR DESCRIPTION
**Pull request by fatima99s**

### CODE REVIEW

File: plugins/aws/s3/bucketEnforceEncryption.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = {
             default: ''
         }
     },
+    realtime_triggers: ['s3:CreateBucket' , 's3:PutBucketPolicy'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/bucketEnforceEncryption.js:26]: delete bucket policy?

File: plugins/aws/s3/bucketLifecycleConfiguration.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     recommended_action: 'Update S3 bucket and create lifecycle rule configuration',
     link: 'https://docs.aws.amazon.com/AmazonS3/latest/dev/how-to-set-lifecycle-configuration-intro.html',
     apis: ['S3:listBuckets', 'S3:getBucketLifecycleConfiguration', 'S3:getBucketLocation'],
+    realtime_triggers: ['s3:CreateBucket', 's3:PutBucketLifeCycleConfiguration'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/bucketLifecycleConfiguration.js:13]: deleteBucketLifecycle

File: plugins/aws/s3/bucketPolicyCloudFrontOac.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ module.exports = {
                 'If an S3 bucket backing a CloudFront distribution does not require the end ' +
                 'user to access the contents through CloudFront, this policy may be violated.'
     },
+    realtime_triggers: ['s3:CreateBucket', 'cloudfront:CreateDistribution', 'cloudfront:UpdateDistribution', 's3:PutBucketPolicy', 's3:DeleteBucketPolicy'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/bucketPolicyCloudFrontOac.js:19]:
```suggestion
    realtime_triggers: ['s3:CreateBucket', 's3:PutBucketPolicy', 's3:DeleteBucketPolicy'],
```

File: plugins/aws/s3/bucketPolicyCloudFrontOai.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ module.exports = {
                 'If an S3 bucket backing a CloudFront distribution does not require the end ' +
                 'user to access the contents through CloudFront, this policy may be violated.'
     },
+    realtime_triggers: ['s3:CreateBucket', 'cloudfront:CreateDistribution', 'cloudfront:UpdateDistribution', 's3:PutBucketPolicy', 's3:DeleteBucketPolicy'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/bucketPolicyCloudFrontOai.js:19]:
```suggestion
    realtime_triggers: ['s3:CreateBucket', 's3:PutBucketPolicy', 's3:DeleteBucketPolicy'],
```

File: plugins/aws/s3/bucketSecureTransportEnabled.js [hunk 5]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     recommended_action: 'Update S3 bucket policy to enforse SSL to secure data in transit.',
     link: 'https://aws.amazon.com/premiumsupport/knowledge-center/s3-bucket-policy-for-config-rule/',
     apis: ['S3:listBuckets', 'S3:getBucketPolicy', 'S3:getBucketLocation'],
+    realtime_triggers: ['s3:CreateBucket', 's3:PutBucketPolicy'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/bucketSecureTransportEnabled.js:13]: delete bucket policy

File: plugins/aws/s3/objectLevelReadEventLogging.js [hunk 6]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     recommended_action: 'Enable object level logging for read events for each S3 bucket.',
     link: 'https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-cloudtrail-logging-for-s3.html#enable-cloudtrail-events',
     apis: ['S3:listBuckets', 'CloudTrail:describeTrails', 'CloudTrail:getEventSelectors', 'S3:getBucketLocation'],
+    realtime_triggers: ['s3:CreateBucket', 'cloudtrail:CreateTrail', 'cloudtrail:PutEventSelectors', 'cloudtrail:PutInsightSelectors'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/objectLevelReadEventLogging.js:12]: What about update bucket

File: plugins/aws/s3/objectLevelWriteEventLogging.js [hunk 7]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     recommended_action: 'Enable object level logging for Write events for each S3 bucket.',
     link: 'https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-cloudtrail-logging-for-s3.html#enable-cloudtrail-events.',
     apis: ['S3:listBuckets', 'CloudTrail:describeTrails', 'CloudTrail:getEventSelectors', 'S3:getBucketLocation'],
+    realtime_triggers: ['s3:CreateBucket', 'cloudtrail:CreateTrail', 'cloudtrail:PutEventSelectors', 'cloudtrail:PutInsightSelectors'],
```
Comment (AkhtarAmir) [inline plugins/aws/s3/objectLevelWriteEventLogging.js:12]: What about update bucket

**Review by AkhtarAmir — CHANGES_REQUESTED** [review]

**Review by AkhtarAmir — APPROVED** [review]

**Review by AkhtarAmir — APPROVED** [review]

**Review by mehakseedat63 — APPROVED** [review]

**Pull request merged**